css/style.css

```
@@ -0,0 +1,3030 @@
+    background-color: #fff;
+    box-shadow: 0 0 30px rgba(0, 0, 0, 0.1);
+    z-index: 999;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+/* 고객 정보 삭제 클릭시*/
+.delete-before-modal-wrap{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    z-index: 999;
+    background-color: rgba(0, 0, 0, 0.3);
+}
+
+.delete-before-modal-wrap.active{
+    display: block;
+}
+
+
+.delete-before-modal{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 500px;
+    height: 500px;
+     border-radius: 15px;
+    box-sizing: border-box;
+    background-color: #fff;
+    box-shadow: 0 0 30px rgba(0, 0, 0, 0.1);
+    z-index: 999;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.delete-before-modal-title{
+    width: 100%;
+    height: 20%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.delete-before-modal-title p{
+    font-size: 1.25rem;
+    color: red;
+    font-weight: 700;
+    margin-top: 10px;
+}
+
+.delete-before-modal-txt{
+    width: 100%;
+    height: 20%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    color: #959595;
+    line-height: 1.5;
+}
+
+.delete-before-agree{
+    width: 100%;
+    height: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+#agree{
+    appearance: none;
+    -webkit-appearance: none;
+    width: 24px;
+    height: 24px;
+    border-radius: 50%;
+    overflow: hidden;
+    background-color: #DBDBDB;
+    border: none;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.1s;
+}
+
+#agree:after{
+    font-family: "Font Awesome 6 Free";
+    content: '\f00c';
+    font-size: 1rem;
+    font-weight: 600;
+    color: #fff;
+}
+
+#agree:hover{
+    background-color: #b5b5b5;
+}
+
+#agree:checked{
+    background-color: #3EDB38;
+}
+
+.delete-before-agree-text{
+    color: #959595;
+    margin-left: 15px;
+}
+
+.delete-before-button{
+    width: 80%;
+    height: 10%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.delete-before-cancel{
+    width: 35%;
+    height: 100%;
+    background-color: transparent;
+     border-radius: 15px;
+    box-sizing: border-box;
+    border: 1px solid #D8D8D8;
+    outline: none;
+    margin-right: 20px;
+    cursor: pointer;
+}
+
+.delete-before-add{
+    width: 35%;
+    height: 100%;
+    background-color: #D8D8D8;
+     border-radius: 15px;
+    box-sizing: border-box;
+    border: none;
+    outline: none;
+    margin-right: 20px;
+    color: #fff;
+    cursor: pointer;
+}
+
+.delete-before-add::after{
+    background-color: #FF0000;
+}
```

이거 이렇게 하면 적용이 안되는디...ㅠ
`input-complete` 처럼 바꾸고 동의 체크박스 클릭할 때 class 값 추가하는 방식으로 갈께~
삭제 버튼에 delete-button id 값도 추가할게!
```suggestion
.delete-before-add.after{
    background-color: #FF0000;
}
```

---

아, cursor 모양도 추가할게
```suggestion
.delete-before-add.agree {
    background-color: #FF0000;
    cursor: pointer;
}
```

위에 .agree 없는 건 `cursor: default;`로 바꿀게~